tests/10apidoc/01register.pl

```diff
@@ -136,6 +136,17 @@ sub local_user_fixture
             )->then_done( $user );
          })->then_with_f( sub {
             my $f = shift;
+            return $f unless defined( my $avatar_url = $args{avatar_url} );
+
+            my $user = $f->get;
+            do_request_json_for( $user,
```

Comment:
If we were optimising for overall runtime speed we might consider `Future->needs_all` for concurrency here; but this is fine enough to solve the problem initially.